### ISSUE TITLE
Rename variables in the decompiler using F2

### ISSUE THREAD

**Issue by bjorn3** [angr/angr-management#622]
This matches the keybinding used to rename variables in VSCode and probably other editors.

**Comment by ltfish** [comment]
We adopt IDA Pro’s keybindings. A solution to this should be different keybinding presets (IDA Pro, Ghidra, VS Code…).

**Comment by bjorn3** [comment]
I see. Could you show the keybinding in the right click menu at least? I'm don't know what keybindings IDA Pro uses.

**Comment by ltfish** [comment]
Absolutely. It's on my TODO list. You are not the first one complaining about it :)
